doc/reference/reference_lua/log.rst

```diff
@@ -81,84 +94,60 @@ Below is a list of all ``log`` functions.
               verbose(message)
               debug(message)
 
-    Output a user-generated message to the :ref:`log file <cfg_logging-log>`,
-    given log_level_function_name = ``error`` or ``warn`` or ``info`` or
-    ``verbose`` or ``debug``.
+    Logs a message with the specified logging level.
+    You can learn more about the available levels from the
+    :ref:`log_level <cfg_logging-log_level>` property description.
+
+    The example below shows how to log a message with the ``info`` level:
+
+    .. code-block:: lua
+
+        log = require('log')
+        log.info('Hello, world!')
 
-    As explained in the description of the configuration setting for
-    :ref:`log_level <cfg_logging-log_level>`, there are seven levels of detail:
+    :param any message:    A log message.
 
-    * 1 – ``SYSERROR``
-    * 2 – ``ERROR`` -- this corresponds to ``log.error(...)``
-    * 3 – ``CRITICAL``
-    * 4 – ``WARNING``  -- this corresponds to ``log.warn(...)``
-    * 5 – ``INFO`` -- this corresponds to ``log.info(...)``
-    * 6 – ``VERBOSE``  -- this corresponds to ``log.verbose(...)``
-    * 7 – ``DEBUG`` -- this corresponds to ``log.debug(...)``
+                           * A message can be a string.
 
-    For example, if ``box.cfg.log_level`` is currently 5 (the default value),
-    then ``log.error(...)``, ``log.warn(...)`` and ``log.info(...)`` messages
-    will go to the log file. However, ``log.verbose(...)`` and
-    ``log.debug(...)`` messages will not go to the log file, because they
-    correspond to higher levels of detail.
+                           * A message may contain C-style format specifiers ``%d`` or ``%s``. Example:
 
-    :param any message:    Usually a string.
+                             .. code-block:: lua
 
-                           Messages may contain C-style format specifiers %d or
-                           %s, so :samp:`log.error('...%d...%s', {x}, {y})`
-                           will work if ``x`` is a number and ``y`` is a string.
+                                 box.cfg{}
+                                 log = require('log')
+                                 log.info('Info %s', box.info.version)
 
-                           Less commonly, messages may be other scalar data types,
-                           or even tables. So :code:`log.error({'x',18.7,true})`
-                           will work.
+                           * A message may be a scalar data type or a table. Example:
+
+                             .. code-block:: lua
+
+                                 log = require('log')
+                                 log.error({500,'Internal error'})
 
     :return: nil
 
     The actual output will be a line in the log, containing:
 
     * the current timestamp,
     * a module name,
-    * 'E', 'W', 'I', 'V' or 'D' depending on ``log_level_function_name``, and
+    * 'E', 'W', 'I', 'V' or 'D' depending on the called function, and
```

Comment:
"and" seems awkward to me here

Reply:
agree, didn't catch it when reviewing old docs